src/unittest/unittestScripts.ts

```diff
@@ -12,6 +12,22 @@ import base64
 import json
 import traceback
 
+from pathlib import Path
+start_dir = sys.argv[2]
```

Comment:
This extension still supports Python 2.7 and pathlib was added in 3.4. Can you please either make it compatible or somehow skip Django for 2.7? It shouldn't fail existing tests (see CI).

Reply:
I'll change the code to fix this